examples/agent/react_agent.ts

```diff
@@ -62,18 +62,18 @@ async function main() {
   });
 
   // Create an ReActAgent with the function tools
-  const agent = new ReActAgent({
+  const agent = new ReACTAgent({
```

Comment:
@himself65 I feel like going from `ReActAgent` to `ReACTAgent` would be considered a breaking change. Given llamaindex is still not v1, and if semantic versioning is being used, possibly a minor bump would have been warranted instead of a patch.

Reply:
Thanks for reviewing, this is unintended. It's okay to revert the naming back